src/qt/modaloverlay.cpp

```diff
@@ -170,3 +189,21 @@ void ModalOverlay::closeClicked()
     showHide(true);
     userClosed = true;
 }
+
+void ModalOverlay::pauseClicked()
+{
+    Q_EMIT requestVerificationPauseOrResume();
+}
+
+void ModalOverlay::setPauseResumeState(bool pauseActive)
+{
+    verificationPauseActive = pauseActive;
+    updatePauseState(pauseActive);
+}
+
+void ModalOverlay::updatePauseState(bool pauseActive)
+{
+    ui->labelNumberBlocksRequested->setText((pauseActive ? "Finish downloading blocks": "Blocks requested from peers"));
+    ui->pauseResumeVerification->setText((pauseActive ? "Resume downloading blocks ": "Pause downloading blocks"));
+    ui->infoLabel->setText((pauseActive && getAmountOfBlocksInFlight() > 0 ? "Wait to finish current downloads...": ""));
```

Comment:
I don't know if you plan to show other messages for this label, but if not then toggle visibility like above?

Reply:
I think this is fine how it is, as long as `nBlocksInFlight > 0` we need to expect network traffic and therefor should still remind the user with the `"Wait to finish current..."`.